pelicanconf.py

```diff
@@ -30,6 +30,7 @@
 LOGO = 'sadev_500w.png'
 MENUITEMS = [
     ('slack', 'https://join.slack.com/t/sanantoniodevs/shared_invite/MjE2ODI0NDQ4NjkwLTE1MDA5MDgwODctMzcxMmZhNjE0Zg'),
+    ('github', 'https://github.com/SADevs/sadevs.github.io'),
```

Comment:
this should probably just be /SADevs and not the link to the website repository. but i don't have super strong feelsing about it

Reply:
I was more thinking that if someone wanted to see the repo where the code was held, then they'd have direct access. I could do two links: Github org page and repo for the site.